file_paths.md

```diff
@@ -163,39 +163,107 @@ and `pxp-agent`, all running as `LocalSystem` by default. It will not
+        puppet.conf *                         # :config
+        routes.yaml                           # :route_file
+        ssl                                   # :ssldir
+
+    C:\ProgramData\PuppetLabs\pxp-agent
+        modules *                             # stores configuration files for pxp-agent modules
+            pxp-module-puppet.conf *          # configuration file of the pxp module puppet (optionally, to override puppet.bat location)
+        pxp-agent.conf                        # pxp-agent configuration file
+
+    C:\Program Files\Puppet Labs\bin *
+        environment.bat *                     # setup LOAD_PATH
+        puppet.bat *                          # bat file wrappers
+        hiera.bat *
+        mco.bat *
+        facter.bat *
+        puppet_shell.bat *                    # targets for shortcuts
+        run_facter_interactive.bat *
+        run_puppet_interactive.bat *
+
+    C:\Program Files\Puppet Labs\facter *
         facts.d *                             # external facts directory (not pluginsync'ed)
 
-    C:\ProgramData\PuppetLabs\mcollective
+    C:\Program Files\Puppet Labs\mcollective *
         plugins *                             # user installed plugins
-        etc *                                                                same
-            client.cfg
-            facts.yaml
-            server.cfg
-        var *                                                                same
-            log *
-                mcollective.log
 
-    C:\ProgramData\PuppetLabs\puppet
-    -------------------------------------------------------------------------------------------------------
-        cache                                 # :vardir                      C:\ProgramData\PuppetLabs\puppet\var
+    C:\Program Files\Puppet Labs\puppet *     # ruby-puppet root
+        bin *                                 # executables and dlls
+            c_rehash *
+            catstomp *
+            catstomp.bat *
+            curl.exe *
+            curl-config *
+            daemon.bat ?
+            environment.bat ?
+            erb.bat *
+            extlookup2hiera *
+            extlookup2hiera.bat *
+            facter.exe *
+            gem.bat *
+            hiera *                           # ruby bin wrapper
+            hiera.bat * (should this be moved?)
+            irb.bat *
+            libcpp-pcp-client.so *
+            libcurl-4.dll *                   # curl dll
+            libeay32.dll *                    # openssl dll
+            libfacter.so *
+            mco *                             # ruby bin wrapper
+            mco.bat * (should this be moved?)
+            mcollectived *                    # ruby bin wrapper
+            minitar *
+            minitar.bat *
+            openssl.exe *
+            puppet *                          # ruby bin wrapper
+            puppet.bat * (should this be moved?)
+            puppetres.dll ?
```

Comment:
Currently comes from puppet: https://github.com/puppetlabs/puppet/tree/master/ext/windows/eventlog. It would be great to move that elsewhere and make it available on buildsources.